modules/custom/entity_access_by_field/entity_access_by_field.install

```diff
@@ -47,7 +47,7 @@ function entity_access_by_field_update_8003() {
       foreach ($data['bundles'] as $bundle) {
         $config_factory
           ->getEditable("field.field.{$entity_type_id}.{$bundle}.{$field_name}")
-          ->set('default_value', NULL)
+          ->set('default_value', [])
```

Comment:
The update hook you're changing here was [added 4 years ago](https://github.com/goalgorilla/open_social/commit/f71cc2eb7bb9c5875a6ec10e0090d932313fa627) That's not going to run on any platform. This is not how update hooks work. Please add a new update hook instead.